### PR TITLE
[FIX] MessagesView title not working

### PR DESCRIPTION
**Pull request by EzequielDeOliveira**
@RocketChat/ReactNative

This pr fix the way to set the title in RoomActionsView items that are wrong after update of react navigation.

**Problem**
Screenshot from 2020-07-16 00-02-59 *Files*

Screenshot from 2020-07-16 00-03-51 *Pinned*

**Resolved**

Screenshot from 2020-07-16 00-05-34

Screenshot from 2020-07-16 00-05-51

### CODE REVIEW

File: app/views/MessagesView/index.js [hunk 1]
```diff
@@ -46,6 +42,7 @@ class MessagesView extends React.Component {
 
 	componentDidMount() {
 		this.load();
+		this.setHeader();
```
Comment (djorkaeffalexandre) [inline app/views/MessagesView/index.js:45]: Should be called on constructor or maybe can cause a header blink.

File: app/stacks/MasterDetailStack/index.js [hunk 2]
```diff
@@ -139,7 +139,6 @@ const ModalStackNavigator = React.memo(({ navigation }) => {
 				<ModalStack.Screen
 					name='MessagesView'
 					component={MessagesView}
-					options={MessagesView.navigationOptions}
```
Comment (djorkaeffalexandre) [inline app/stacks/MasterDetailStack/index.js:142]: Why did you remove this from the tablet mode but keep it on the regular inside stack?

File: app/views/MessagesView/index.js (hunk 1 above)
Comment (EzequielDeOliveira) [reply app/views/MessagesView/index.js]: Right, I called this in the constructor now.

File: app/stacks/MasterDetailStack/index.js (hunk 2 above)
Comment (EzequielDeOliveira) [reply app/stacks/MasterDetailStack/index.js:142]: Removed.
